test/multiple-inputs.ll

```diff
@@ -0,0 +1,42 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-as %s -o %t1.bc
+; RUN: llvm-spirv %t.bc %t1.bc -spirv-text
```

Comment:
Please add test for `-o` option in multi-input mode

Reply:
Added.